packages/core/src/index.js

```diff
@@ -37,6 +39,27 @@ function createConfig (webpack, configSetters) {
 }
 
 /**
```

Comment:
I think we should move the version-parsing function into a separate file. It's not big, but it's also not related to the rest of the file's code, it's something completely independent.